ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/entry/AuditEntry.java

```diff
@@ -107,14 +111,22 @@ public Status getStatus()
         return status;
     }
 
+    /**
+     * @return the timestamp when this entry was created. Represented by the number of milliseconds since Epoch.
+     */
+    public long getTimestamp()
+    {
+        return timestamp;
+    }
+
     /**
      * Create a new {@link Builder} instance.
      *
      * @return a new instance of {@link Builder}.
      */
     public static AuditEntry.Builder newBuilder()
     {
-        return new Builder();
+        return new Builder(System.currentTimeMillis());
```

Comment:
I'd prefer if the timestamp was set explicitly on the builder instead.
- If we start to add parameters to the builder ctor, then to be consistent we should add other "mandatory" parameters as well. But that would defeat the purpose of a builder.
- Also, builder instances are already today created with some "ugly magic" in the AuditEntryBuilderFactory class. With  the implicit timestamp we would add a little more magic and thereby make it less obvious of where the information is coming from.
- And perhaps most importantly, in some cases we create two AuditEntry instances - one for the ATTEMPT and another for the possible FAILURE. Since both entries represent the same request they should have the same timestamp. So in practice I think we need to create the timestamp in the AuditQueryHandler class and pass it to the AuditAdapter. Same thing for authentications. Less elegant, but more accurate IMO.